File: src/main/java/net/xalcon/technomage/client/gui/multiblocks/GuiBrickFurnace.java
package net.xalcon.technomage.client.gui.multiblocks;

import net.minecraft.client.gui.inventory.GuiContainer;
import net.minecraft.entity.player.EntityPlayer;
import net.xalcon.technomage.common.container.ContainerBrickFurnace;

public class GuiBrickFurnace extends GuiContainer
{
    public GuiBrickFurnace(EntityPlayer player)
    {
        super(new ContainerBrickFurnace(player));
    }

    @Override
    protected void drawGuiContainerBackgroundLayer(float partialTicks, int mouseX, int mouseY)
    {
    }
}

File: src/main/kotlin/net/xalcon/technomage/client/gui/multiblocks/GuiBrickFurnace.kt
package net.xalcon.technomage.client.gui.multiblocks

import net.minecraft.client.gui.inventory.GuiContainer
import net.minecraft.entity.player.EntityPlayer
import net.xalcon.technomage.common.containers.multiblock.ContainerBrickFurnace

class GuiBrickFurnace(player: EntityPlayer) : GuiContainer(ContainerBrickFurnace(player))
{
    override fun drawGuiContainerBackgroundLayer(partialTicks: Float, mouseX: Int, mouseY: Int)
    {
    }
}